### PR TITLE
feat(tutorial): complete step 1

### PR DESCRIPTION
**Pull request by daka1510**

### CODE REVIEW

**Comment by ZrianinaMariia** [comment]
Hey @daka1510 👍 Almost there! Please remove the Button component. 
image

**Comment by daka1510** [comment]
Thanks for your quick review, @ZrianinaMariia. Done :).

File: .idea/carbon-tutorial.iml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (ZrianinaMariia) [inline .idea/carbon-tutorial.iml]: Just for future - you could add .idea folder to .gitignore 👍

**Comment by ZrianinaMariia** [comment]
@daka1510 Looks good 🥇 ! I've recorded your completion of step 1.

**Pull request closed without merging**